test cases/common/132 dependency file generation/main .c

```diff
@@ -0,0 +1,5 @@
+int
+main(int argc, char *argv[])
+{
```

Comment:
This is against all coding standards we have. It should be:

    int main(int argc, char **argv) {